source/deployment/cluster.rst

```diff
@@ -1,116 +1,126 @@
-High Availability (E20) - Beta
+High Availability Cluster (E20)
 ===============================
 
 *Available in Enterprise Edition E20.*
 
-High Availability support enables a Mattermost server to scale and maintain service during outages and hardware failures through the use of redundant infrastructure. 
+The high availability cluster feature enables a Mattermost system to maintain service during outages and hardware failures through the use of redundant infrastructure.
 
-- Support for high availability database configurations is available in Mattermost Enterprise Edition E20 version 3.1 and higher. 
-- Support for high availability configurations of the Mattermost application server is available in Mattermost Enterprise Edition E20 version 3.3 and higher. 
+High availability in Mattermost consists of running redundant Mattermost application servers, redundant database servers, and redundant load balancers. The failure of any one of these components does not interrupt operation of the system.
 
-This documentation provides a deployment guide, configuration and compatibility details, as well as troubleshooting advice.
+.. contents::
+  :backlinks: top
+  :local:
+  
+Continuous Operation During Server Updates
```

Comment:
Propose calling this section "Continuous Operation Overview" and laying out the requirements for continuous operation similar to how Ian had proposed. Right now it doesn't feel like there is a good flow to this section.

![image](https://cloud.githubusercontent.com/assets/14333569/22491677/6d12712a-e7e2-11e6-9cc9-fed0bf735eab.png)